package.json

```diff
@@ -35,24 +35,27 @@
     "@storyblok/react": "^2.4.0",
     "cnbuilder": "^3.1.0",
     "framer-motion": "^10.15.0",
-    "netlify-plugin-vault-variables": "^1.0.7",
-    "next": "^13.4.9",
-    "node-vault": "^0.9.22",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "next": "13.4.9",
     "storyblok-rich-text-react-renderer-ts": "^3.2.0"
   },
+  "peerDependencies": {
+    "react": "^18.2",
+    "react-dom": "^18.2"
+  },
   "devDependencies": {
     "@tailwindcss/container-queries": "^0.1.1",
     "@types/node": "^20.4.7",
-    "@types/react": "^18.2.18",
-    "@types/react-dom": "^18.2.7",
+    "@types/react": "^18.2",
+    "@types/react-dom": "^18.2",
     "autoprefixer": "^10.4.14",
     "decanter": "^7.1.1",
     "eslint": "^8.50.0",
     "eslint-config-next": "^13.4.13",
-    "postcss": "^8.4.31",
-    "tailwindcss": "^3.3.3",
-    "typescript": "^5.1.6"
+    "netlify-plugin-vault-variables": "^1.0.7",
+    "postcss": "^8.4.27",
```

Comment:
Any particular reason that postcss went from ^8.4.31 to ^8.4.27? I'm guess this might be from merging in `dev` (I might've updated this to .31 at some point).

Reply:
With the version constraint, those patch versions won't matter as they will still pull the latest patch. Not sure where that difference snunk in.